### PR TITLE
[Tool] SQL Testing Framework

### PR DESCRIPTION
**Pull request by andyziye**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

Add SQL-tester framework

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+### 3. Requirements
+Information about the dependencies is stored in the `requirements.txt`.   
+You can install them by:  
+```shell
+$ python3 -m pip install -r requirements.txt
+```
+
+### 4. Config
+First you need to prepare a target sr to be tested, and then store their information in `conf/sr.conf`.
```
Comment (LiShuMing) [inline test/README.md:42]: Is that possible to manage sr's fe/be process by using python scripts ? so can test more failover screnes.

Comment (AndyZiYe) [reply test/README.md:42]: Yes, we already support this type of capability in the integration testing framework. Whether or not to inherit into the SQL framework will be considered in the future.

**Comment by trueeyu** [comment]
Remove test/log/sql_test.log?

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+skip_res_cmd = [
+    "${SKIP_SQL_REG_1}",
+    "${SKIP_SQL_REG_2}"
+]
+```
+
+#### 1.2 SKIP FLAG
+You can skip the check round by adding the **`[UC]`** flag in front of the statement that needs to skip the check.  
+Compared to `SKIP ARRAY`, `[UC]` is more flexible as it affects a single case without affecting other cases.
+```clickhouse
```
Comment (trueeyu) [inline test/README.md:203]: clickhouse?

Comment (AndyZiYe) [reply test/README.md:203]: That's a md grammar. I'll rm them.

**Pull request merged**